static/styles/zulip.scss

```diff
@@ -2656,7 +2651,7 @@ select.inline_select_topic_edit {
 
     .top-navbar-border {
         margin-left: 40px;
-        width: calc(100% - 116px);
+        width: calc(100% - 108px);
```

Comment:
Could we extract a set of variables used elsewhere that add up to 108px to use here?

Reply:
We can by setting the middle section width(900px) to a variable and reuse that to calculate the width for other devices

But that I guess to be done in a separate PR. Right now we use `margin` to adjust width of `column-middle`